app/src/main/java/com/example/tour_a_versity_v3/MapsActivity.kt

```diff
@@ -60,12 +63,12 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnInfoWi
         )
     }
     override fun onInfoWindowClick(p0: Marker?) {
-        if (p0 == tucMarker){
-            val intent = Intent(this, TUC_Info::class.java)
+        if (p0 === tucMarker){
```

Comment:
In Kotlin there a difference between structural equal operator (==) and referential operator (===). Made this change per suggestion here: https://agrawalsuneet.github.io/blogs/equality-in-kotlin/#:~:text=Structural%20Equality%20('%3D%3D')&text=%3D%3D%20operator%20in%20Kotlin%20only,not%20equal%20to%20each%20other.

Reply:
This is an excellent reference on == vs ===!